db/db_block_cache_test.cc

```diff
@@ -1290,12 +1290,14 @@ TEST_F(DBBlockCacheTest, CacheEntryRoleStats) {
   int iterations_tested = 0;
   for (bool partition : {false, true}) {
     for (std::shared_ptr<Cache> cache :
-         {NewLRUCache(capacity), NewClockCache(capacity),
-          NewFastLRUCache(capacity)}) {
+         {NewLRUCache(capacity), NewClockCache(capacity)}) {
+      // This test doesn't support FastLRUCache because the
```

Comment:
Eventually we need to support NewFastLRUCache here. Please add a task  and add a TODO here.